### PR TITLE
Update go-tspi

### PR DESCRIPTION
**Pull request by mjg59**
We need to ship the tpmown binary in order to allow admins to configure their TPMs properly, and we need a couple of fixes to the tpmd systemd unit.

### CODE REVIEW

File: app-crypt/go-tspi/go-tspi-9999.ebuild [hunk 1]
```diff
@@ -21,10 +21,10 @@ DEPEND="${RDEPEND}"
 
 src_compile() {
 	      go_build "${COREOS_GO_PACKAGE}/tpmd"
+      	      go_build "${COREOS_GO_PACKAGE}/tpmown"
 }
 
 src_install() {
 	dobin ${GOBIN}/*
 	systemd_dounit "${FILESDIR}"/tpmd.service
-	systemd_enable_service multi-user.target tpmd.service
```
Comment (marineam) [inline app-crypt/go-tspi/go-tspi-9999.ebuild]: Is there anything else that will activate tpmd.service or will users need to enable it now?

Comment (mjg59) [inline app-crypt/go-tspi/go-tspi-9999.ebuild]: Right now we're asking users to enable it explicitly on DTC setups

**Comment by marineam** [comment]
LGTM

**Comment by mischief** [comment]
@mjg59 please put a note in the manifest at least about the introduction of tpmown.

**Pull request merged**